.github/workflows/ci.yaml

```diff
@@ -86,46 +128,36 @@ jobs:
           restore-keys: |
             gobuild-linux-amd64
 
-      - name: Checkout charts repository for deployments
-        uses: actions/checkout@v2
-        with:
-          repository: newrelic/helm-charts
-          path: helm-charts-newrelic
-
-      - name: Install kind
-        run: go install sigs.k8s.io/kind@v0.11.1
-
       - name: Create k8s Kind Cluster
-        run: make -C newrelic-k8s-metrics-adapter kind-up
+        run: make kind-up
 
       - name: Run integration tests
         env:
           NEWRELIC_API_KEY: "${{ secrets.NEWRELIC_API_KEY }}"
           NEWRELIC_ACCOUNT_ID: "${{ secrets.NEWRELIC_ACCOUNT_ID }}"
-        run: make -C newrelic-k8s-metrics-adapter test-integration
+        run: make test-integration
 
       - name: Build Docker image
-        run: make -C newrelic-k8s-metrics-adapter image-push IMAGE_REPO=localhost:5000/newrelic-k8s-metrics-adapter
+        run: make image-push IMAGE_REPO=localhost:5000/newrelic-k8s-metrics-adapter
 
       - name: Create Helm release
         env:
-          KUBECONFIG: ./newrelic-k8s-metrics-adapter/kubeconfig
+          KUBECONFIG: ./kubeconfig  # Generated by `make kind-up`
           NEWRELIC_API_KEY: "${{ secrets.NEWRELIC_API_KEY }}"
           NEWRELIC_ACCOUNT_ID: "${{ secrets.NEWRELIC_ACCOUNT_ID }}"
         run: |
-          make -C newrelic-k8s-metrics-adapter generate-local-values
-          helm install newrelic-k8s-metrics-adapter ./helm-charts-newrelic/charts/newrelic-k8s-metrics-adapter \
-          --values ./newrelic-k8s-metrics-adapter/values-dev.yaml \
-          --values ./newrelic-k8s-metrics-adapter/values-local.yaml \
+          make generate-local-values
+          helm install newrelic-k8s-metrics-adapter ./charts/newrelic-k8s-metrics-adapter \
           --set image.repository=localhost:5000/newrelic-k8s-metrics-adapter \
           --set image.pullPolicy=IfNotPresent \
-          --wait
+          --values values-dev.yaml --values values-local.yaml \
+          --wait --timeout 240s
 
       - name: Run e2e tests
-        run: make -C newrelic-k8s-metrics-adapter test-e2e
+        run: make test-e2e
 
-  lint:
```

Comment:
in nri-kubernetes we are adding as well:
```
      - uses: newrelic/newrelic-infra-checkers@v1
      - name: Semgrep
        uses: returntocorp/semgrep-action@v1
        with:
          auditOn: push
```

Reply:
Solved in [#5fe777a](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/5fe777afae5b643f5965f056738b130343777573)

Reply:
I fear something weird happened, I am not able to see this commit

Reply:
I saw you forced-pushed, but I cannot see any code related in last commit

Reply:
Because Github was having issues with the pipelines and I give up. I am pushing it right now.

Solved in [#eeb8005](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/eeb80051425d0384b77c263028c6f08786340e21)